CSS/css_completions.py

```diff
@@ -87,16 +88,10 @@ def on_query_completions(self, view, prefix, locations):
             return sublime.CompletionList(items)
         return None
 
-    def complete_property_name(self, view, prefix, pt):
+    def complete_property_name(self, view: sublime.View, prefix: str, pt: int) -> List[sublime.CompletionItem]:
         text = view.substr(sublime.Region(pt, view.line(pt).end()))
         matches = self.re_value.search(text)
-        if matches:
-            colon, space, value, term = matches.groups()
-        else:
-            colon = ""
-            space = ""
-            value = ""
-            term = ""
+        colon, space, value, term = matches.groups() if matches else ("", "", "", "")
```

Comment:
This change is orthogonal, IMHO. 

I prefer the more explicit former form as it doesn't require unpacking to be applied in case of no match.